core/src/main/java/org/stellar/anchor/sep1/Sep1Service.java

```diff
@@ -46,6 +45,17 @@ public Sep1Service(Sep1Config sep1Config) throws IOException, InvalidConfigExcep
     }
   }
 
+  private String fetchTomlFromURL(String url) throws IOException {
+    try {
+      return NetUtil.fetch(url);
+    } catch (IOException e) {
+      // Obfuscate the message and rethrow
+      String obfuscatedMessage = String.format("Unable to fetch data from %s", url);
+      debugF(obfuscatedMessage); // Log the obfuscated message using the debugF method
```

Comment:
If response is not successful this will produce 3 debug logs with the same message

Reply:
ah yea, im logging at each layer. let me look at that. thanks!